packages/nodes-base/nodes/Microsoft/Sql/test/utils.test.ts

```diff
@@ -0,0 +1,130 @@
+import mssql from 'mssql';
+import {
+	configurePool,
+	deleteOperation,
+	insertOperation,
+	updateOperation,
+} from '../GenericFunctions';
+
+jest.mock('mssql', () => {
+	const originalModule = jest.requireActual('mssql');
+	const { Request } = originalModule;
+	Request.prototype.query = jest.fn(async function () {
```

Comment:
can we please also mock `.input` calls, and asserts that the values are correct s well, and not just the templated queries 🙏🏽 ?

Reply:
@netroy 
updated tests with values checks